python/cugraph/cugraph/dask/sampling/uniform_neighbor_sample.py

```diff
@@ -396,44 +452,30 @@ def uniform_neighbor_sample(
     else:
         indices_t = numpy.int32
 
-    start_list = start_list.rename(start_col_name)
-    if batch_id_list is not None:
-        batch_id_list = batch_id_list.rename(batch_col_name)
-        if hasattr(start_list, "compute"):
-            # mg input
-            start_list = start_list.to_frame()
-            batch_id_list = batch_id_list.to_frame()
-            ddf = start_list.merge(
-                batch_id_list,
-                how="left",
-                left_index=True,
-                right_index=True,
-            )
-        else:
-            # sg input
-            ddf = cudf.concat(
-                [
-                    start_list,
-                    batch_id_list,
-                ],
-                axis=1,
-            )
-    else:
+    if hasattr(start_list, "to_frame"):
+        start_list = start_list.rename(start_col_name)
         ddf = start_list.to_frame()
+    else:
+        ddf = start_list
+        columns = ddf.columns
+        ddf = ddf.rename(
+            columns={columns[0]: start_col_name, columns[-1]: batch_col_name}
+        )
 
     if input_graph.renumbered:
         ddf = input_graph.lookup_internal_vertex_id(ddf, column_name=start_col_name)
 
-    if hasattr(ddf, "compute"):
-        ddf = get_distributed_data(ddf)
-        wait(ddf)
-        ddf = ddf.worker_to_parts
-    else:
-        splits = cp.array_split(cp.arange(len(ddf)), len(Comms.get_workers()))
-        ddf = {w: [ddf.iloc[splits[i]]] for i, w in enumerate(Comms.get_workers())}
-
     client = get_client()
     session_id = Comms.get_session_id()
+    n_workers = get_n_workers()
+
+    if not hasattr(ddf, "compute"):
```

Comment:
I think we should use  `isinstance` to be cleaner. 
```suggestion
    if isinstance(ddf, cudf.DataFrame):
```